pkgs/servers/sql/mariadb/default.nix

```diff
@@ -98,28 +96,24 @@ common = rec { # attributes common to both builds
 client = stdenv.mkDerivation (common // {
   name = "mariadb-client-${common.version}";
 
-  outputs = [ "bin" "dev" "out" ];
+  outputs = [ "dev" "out" "man" ];
 
   propagatedBuildInputs = [ openssl zlib ]; # required from mariadb.pc
 
+  patches = [ ./cmake-plugin-includedir.patch ];
+
   cmakeFlags = common.cmakeFlags ++ [
     "-DWITHOUT_SERVER=ON"
+    "-DWITH_WSREP=OFF"
   ];
 
-  preConfigure = common.preConfigure + ''
-    cmakeFlags="$cmakeFlags \
-      -DINSTALL_BINDIR=$bin/bin \
-      -DINSTALL_SCRIPTDIR=$bin/bin \
-      -DINSTALL_SUPPORTFILESDIR=$bin/share/mysql \
-      -DINSTALL_DOCDIR=$bin/share/doc/mysql \
-      -DINSTALL_DOCREADMEDIR=$bin/share/doc/mysql \
-      "
-  '';
-
-  # prevent cycle; it needs to reference $dev
-  postInstall = common.postInstall + ''
-    moveToOutput bin/mysql_config "$dev"
-    moveToOutput bin/mariadb_config "$dev"
+  postInstall = ''
+    rm -r "$out"/share/mysql
+    rm -r "$out"/share/doc/mysql/{policy,systemd}
+    rm "$out"/bin/{galera_new_cluster,galera_recovery,mariadb-service-convert,msql2mysql,my_print_defaults,mysql_convert_table_format,mysqld_safe_helper,mysql_install_db,mysql_plugin,mysql_secure_installation,mysql_setpermission,mysql_upgrade,mytop,perror,replace,resolveip,resolve_stack_dump,wsrep_sst_rsync_wan,mysql_config,mariadb_config}
```

Comment:
Can you wrap or split long lines for readability?

Reply:
Fixed